src/main/scala/swiss/dasch/api/ProjectsEndpointsHandler.scala

```diff
@@ -173,7 +173,9 @@ final case class ProjectsEndpointsHandler(
     .serverLogic(principal => { case (shortcode, filenames, stream) =>
       for {
         _ <- authorizationHandler.ensureAdminScope(principal)
-        _ <- bulkIngestService.uploadSingleFile(shortcode, filenames, stream).mapError(InternalServerError(_))
+        _ <- bulkIngestService.uploadSingleFile(shortcode, filenames, stream).mapError { e =>
+               InternalServerError(e.getOrElse(Throwable("ingest in progress")))
```

Comment:
*issue* This should return 409 in case the build-ingest is running, i.e. the `e` is a `None`

Like so:
```scala
.mapError {
  case None                        => failBulkIngestInProgress(shortcode)
  case Some(e: Exception) => InternalServerError(e)
}
```
              
See the mapping csv endpoint for example.